packages/hash/frontend/src/pages/[account-slug]/types/entity-type/property-list-card.tsx

```diff
@@ -165,14 +165,10 @@ const InsertPropertyRow = ({
               </IconButton>
             </Box>
             <PropertyTypeForm
-              createButtonProps={withHandler(
-                bindToggle(modalPopupState),
-                () => {
-                  // onAdd();
-                },
-              )}
+              createButtonProps={{}}
```

Comment:
If `createButtonProps` is not planned to be used anymore, we can remove it I guess?

Reply:
[`8991f77` (#1345)](https://github.com/hashintel/hash/pull/1345/commits/8991f77b45db4464fa3d0731cb73a5401299e4fa)